platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/camera/CameraPositionActivity.java

```diff
@@ -28,6 +28,7 @@
 
   private MapView mapView;
   private MapboxMap mapboxMap;
```

Comment:
That is a requirement for our autogenerated tests, see https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/utils/OnMapReadyIdlingResource.java#L43 where we use reflection on the mapboxMap field. In this specific case we could easily make it accessed by mapboxMap invocations below.